tasmota/tasmota_xdrv_driver/xdrv_129_rddl_network.ino

```diff
@@ -132,6 +132,10 @@ bool claimNotarizationMutex()
   else
     return false;
 }
+bool executePoP = false;
+void SetExecutePoP(){
+  executePoP = true;
+}
```

Comment:
Do we want to add the capability to opt out of Pop as well? Currently you can just activate it. 

Bit this can be done in a separate PR